### PR TITLE
Bugfix/ui changes

### PR DESCRIPTION
**Pull request by david-forster10**

### CODE REVIEW

File: src/components/Shared/ViewsforProjectModal/ProjectsLayout.jsx [hunk 1]
```diff
@@ -6,10 +6,10 @@ import '../css/ProjectForm.css';
 export class ProjectsLayout extends React.Component {
 	handleFormUpdate = () => {
 		let newProjectData = {
-			title: this.projectTitle.value,
-			studyCode: this.projectStudyCode.value,
-			E2EManager: this.projectEndToEndManager.value,
-			status: this.projectStatus.value,
+			projectTitle: this.projectTitle.value,
```
Comment (gwolverson) [inline src/components/Shared/ViewsforProjectModal/ProjectsLayout.jsx]: Is there a reason the namings for all these have changed? Will this affect the expected values on the server side?

File: src/components/Shared/ViewsforProjectModal/RequestsLayout.jsx [hunk 2]
```diff
@@ -34,24 +34,24 @@ export class RequestsLayout extends React.Component {
...
 								<Form horizontal>
-									<FieldGroup id='requestTitle' onChange={this.handleFormUpdate} type='text' label='Title:' placeholder='' inputRef={ ref => {
-										this.requestTitle = ref;
+									<FieldGroup id='requestTitle' onChange={this.handleFormUpdate} type='text' label='Title:' placeholder='' inputRef={(ref2) => {
+										this.requestTitle = ref2;
 									}}/>
 									<br />
-									<FieldGroup id='requestCode' onChange={this.handleFormUpdate} type='text' label='Request Code:' placeholder='' inputRef={ ref => {
-										this.requestCode = ref;
+									<FieldGroup id='requestCode' onChange={this.handleFormUpdate} type='text' label='Request Code:' placeholder='' inputRef={(ref2) => {
```
Comment (gwolverson) [inline src/components/Shared/ViewsforProjectModal/RequestsLayout.jsx]: Reason for ref2 over ref ? (and subsequent ref3 etc

Comment (david-forster10) [reply src/components/Shared/ViewsforProjectModal/RequestsLayout.jsx]: These were used when trying to bug fix to see if a possible cause and weren't reverted. Now reverted

File: src/components/Shared/ViewsforProjectModal/ProjectsLayout.jsx (hunk 1 above)
Comment (david-forster10) [reply src/components/Shared/ViewsforProjectModal/ProjectsLayout.jsx]: These were changed to match expecting values a level higher, they are put into correctly named variables that match the backend in ProjectForm.jsx

**Pull request merged**